packages/contracts-bedrock/src/governance/ProposalValidator.sol

```diff
@@ -227,6 +251,98 @@ contract ProposalValidator is OwnableUpgradeable, ReinitializableBase, ISemver {
         transferOwnership(_owner);
     }
 
+    /// @notice Submits a GovernanceFund or CouncilBudget proposal type that transfers OP tokens for approval and
+    /// voting.
+    /// @param _criteriaValue Since the passing criteria type is always "Threshold", this value will be the percentage
+    /// that will be used to calculate the fraction of the votable supply that the proposal will need in votes in order
+    /// to pass.
+    /// @param _optionsDescriptions The strings of the different options that can be voted.
+    /// @param _optionsRecipients An address for each option to transfer funds to in case the option passes the voting.
+    /// @param _optionsAmounts The amount to transfer for each option in case the option passes the voting.
+    /// @param _description Description of the proposal.
+    /// @param _proposalType The type of proposal (must be GovernanceFund or CouncilBudget).
+    /// @return proposalHash_ The hash of the submitted proposal.
+    function submitFundingProposal(
+        uint128 _criteriaValue,
```

Comment:
Thought about this a bit more, I believe we should stick with the specs, the reason we use the percentage here is because since this function is permissionless we want the user to have an easier experience on providing these data. The idea is that they provide the percentage and we use that to calculate the amount of tokens **relative** to the existing voting supply